### PR TITLE
Added aop23_from_script_052323.ttl

### PR DESCRIPTION
**Pull request by mam288**
Addresses issues #1  and #2

### CODE REVIEW

**Comment by mam288** [comment]
Added .owl and .ofn files for AOP23

**Review by balhoff — CHANGES_REQUESTED** [review]
@mam288 a few change requests that we should probably put in a specs document:
- Only commit turtle format files here, one file for each model.
- Put all models in a `models` folder under the project root
- Make sure each model declares an ontology IRI, using a triple like this: `<https://noctua.apps.renci.org/model/some_identifier> a owl:Ontology`
- Make sure the filename matches the local part of the ontology IRI, like `some_identifier.ttl`
- Don't include any `owl:imports` triples.

Let me know if you have any questions!

**Comment by mam288** [comment]
@balhoff Will do, thanks Jim! Just to clarify, you only want the .ttl file? You don't want me to use robot to convert them to .owl or .opn files?

**Comment by balhoff** [comment]
Right, the robot conversion was just to validate parseability for you.

**Comment by mam288** [comment]
Ahh I see. Thanks! 
Also for the IRI statement I use Noctua's address and then use an identifier (AOP_23), right?  (see below)
<https://noctua.apps.renci.org/model/AOP_23>. 
I should then name the matching .ttl file AOP_23.ttl?

**Comment by balhoff** [comment]
> Ahh I see. Thanks!
> Also for the IRI statement I use Noctua's address and then use an identifier (AOP_23), right? (see below)
> https://noctua.apps.renci.org/model/AOP_23.
> I should then name the matching .ttl file AOP_23.ttl?

Yes, that's it.

**Comment by mam288** [comment]
@balhoff I added .ttl files for all of the models that ran successfully. Can you just check that they look correct and then upload to Noctua if they do? Thanks!

File: models/aop_100.ttl [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+<http://www.co-ode.org/ontologies/ont.owl#prostaglandin_biosynthetic_process> rdf:type owl:NamedIndividual ,
+	<http://purl.obolibrary.org/obo/GO_0001516> ;
+	<http://purl.obolibrary.org/obo/RO_0002234> :prostaglandin_F2alpha ;
+	<http://purl.obolibrary.org/obo/RO_0002233> :prostaglandin_F2alpha ;
+	<http://purl.obolibrary.org/obo/RO_0000057> :prostaglandin_F2alpha ;
+	<http://purl.obolibrary.org/obo/RO_0002332> :prostaglandin_F2alpha  .
+
+###  http://www.co-ode.org/ontologies/ont.owl#fertility
+<http://www.co-ode.org/ontologies/ont.owl#fertility> rdf:type owl:NamedIndividual ,
+	<http://purl.obolibrary.org/obo/D005298> ;
```
Comment (balhoff) [inline models/aop_100.ttl]: @mam288 just noticed this ID—is it a typo for `DO_`? Can you check for others?

**Review by balhoff — CHANGES_REQUESTED** [review]

File: models/aop_101.ttl [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+###  http://www.co-ode.org/ontologies/ont.owl#mating_pheromone_secretion
+<http://www.co-ode.org/ontologies/ont.owl#mating_pheromone_secretion> rdf:type owl:NamedIndividual ,
+	<http://purl.obolibrary.org/obo/GO_0071834> ;
+	<http://purl.obolibrary.org/obo/RO_0019000> :attraction-related_behavior  .
+
+###  http://www.co-ode.org/ontologies/ont.owl#population_of_organisms
+<http://www.co-ode.org/ontologies/ont.owl#population_of_organisms> rdf:type owl:NamedIndividual ,
+	<http://purl.obolibrary.org/obo/PCO_0000001> ;
+	<http://purl.obolibrary.org/obo/> :population_growth_rate  .
```
Comment (balhoff) [inline models/aop_101.ttl]: @mam288 I think there's a missing part of a relation ID here.

Comment (balhoff) [reply models/aop_101.ttl]: This happens in some other files as well.

**Review by balhoff — CHANGES_REQUESTED** [review]
@mam288 I noticed a couple more issues that maybe we should fix before loading.

**Comment by mam288** [comment]
@balhoff There are some cases where the code couldn't predict a relationship (similar to how some have multiple relationships). I'll have to double check but I think these are those cases. The statement is supposed to be a placeholder. Is there a better way to do this? Thanks!

**Comment by balhoff** [comment]
> @balhoff There are some cases where the code couldn't predict a relationship (similar to how some have multiple relationships). I'll have to double check but I think these are those cases. The statement is supposed to be a placeholder. Is there a better way to do this? Thanks!

How about just using `https://w3id.org/biolink/vocab/related_to`?

**Comment by balhoff** [comment]
@mam288 or if you think it makes sense for the situations you have, you could use the very general [causally related to](http://purl.obolibrary.org/obo/RO_0002410) and at least stick with an RO term.

**Comment by mam288** [comment]
@balhoff I went with causally related term for the edges without an assigned term. I uploaded new .ttl files to the "meisha" branch. Let me know if you have any questions.

**Comment by mam288** [comment]
@balhoff I updated the IRI statements for the MESH terms.

**Review by balhoff — APPROVED** [review]

**Pull request merged**